examples/unresolved-cfd-dem/gas-solid-spouted-cylinder-bed/dem-packing-in-spouted-bed-generator.prm

```diff
@@ -0,0 +1,133 @@
+# Listing of Parameters
+#----------------------
+
+set dimension = 3
+
+#---------------------------------------------------
+# Simulation Control
+#---------------------------------------------------
+
+subsection simulation control
+  set time step        = 0.00001
+  set time end         = 1.00
+  set log frequency    = 1000
+  set output frequency = 10000
+  set output path      = ./output_dem/
+end
+
+#---------------------------------------------------
+# Timer
+#---------------------------------------------------
+
+subsection timer
+  set type = none
+end
```

Comment:
```suggestion
#---------------------------------------------------
# Timer
#---------------------------------------------------

subsection timer
  set type = none
end
```
I would use the iteration timer so users have a feel when the bed is being generated of the time it will take.